Configurations/50-win-arm.conf

```diff
@@ -0,0 +1,35 @@
+# Windows on ARM targets. ARM compilers are additional components in
+# VS2017, i.e. they are not installed by default. And when installed,
+# there are no "ARM Tool Command Prompt"s on Start menu, you have
+# to locate vcvarsall.bat and act accordingly. VC-WIN32-ARM has
+# received limited testing with evp_test.exe on Windows 10 IoT Core,
+# but not VC-WIN64-ARM, no hardware... In other words they are not
+# actually supported...
+#
+# Another thing to keep in mind [in cross-compilation scenario such as
+# this one] is that target's file system has nothing to do with
+# compilation system's one. This means that you're are likely to use
+# --prefix and --openssldir with target-specific values. 'nmake install'
+# step is effectively meaningless in cross-compilation case, though it
+# might be meaningful to 'nmake install DESTDIR=S:\ome\where' where you
+# can point Visual Studio to when compiling custom application code.
+
+my %targets = (
+    "VC-WIN32-ARM" => {
+        inherit_from    => [ "VC-noCE-common" ],
+        defines         => add("_ARM_WINAPI_PARTITION_DESKTOP_SDK_AVAILABLE"),
+        bn_ops          => "BN_LLONG RC4_CHAR EXPORT_VAR_AS_FN",
+        # /NODEFAULTLIB:kernel32.lib is needed, because MSVCRT.LIB has
+        # direct reference to kernel32.lib, but we don't actually want
+        # it in "onecore" build.
+        lflags          => add("/NODEFAULTLIB:kernel32.lib"),
+        ex_libs         => "onecore.lib onecore_downlevel.lib",
```

Comment:
Will onecode_downlevel.lib still be needed when #4965 gets merged?

Reply:
It would be possible to omit it [and add additional define].

Reply:
... but what's "needed" anyway? I can confirm that binaries built as suggested do run on a **certified** Windows IoT device, even though they are linked with onecore_downlevel.lib. [Well, "certified" means that I'm told that it is, not that I certified it myself.] One can argue that doing so make it non-compliant with some "big picture", but you know what? Said "big picture" is not really self-consistent. They say "onecore only, here is the list", but you also end-up with reference to CRT. In other words it's not really that categorical as "onecore only". It might happen that reliance on CRT disqualifies binary code as compliant with "big picture", but then we are out of luck anyway... [In the context you might recall that at some point I actually suggested to consider removal or rather minimization of the reliance on MS CRT, but met no support for the idea.]